client/src/pages/signin/Signin.jsx

```diff
@@ -87,9 +87,9 @@ function Signin() {
         let userMetadata = await magic.user.getMetadata();
         await setUser(userMetadata);
         let newDidToken = await magic.user.getIdToken({ lifespan: 24 * 60 * 60 * 7 });
-        window.localStorage.setItem("didToken", newDidToken);
+        // window.localStorage.setItem("didToken", newDidToken);
```

Comment:
We need to store the didToken in the cookie storage.

Reply:
Yes and we are doing that. `window.localStorage.setItem("didToken", newDidToken);` will save the token in the local storage but we want to store the token in the cookies. So I am sending the token to the server and when response comes back it will automatically set the token in the cookis.